### PR TITLE
feat: remove dao-dao imports, copy structs

### PR DESCRIPTION
**Pull request by apollo-pacman**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ default = ["osmosis"]
 osmosis = []
 
 [dependencies]
-osmosis-std = { git = "https://github.com/osmosis-labs/osmosis-rust", rev = "3ea06eda3e5f2979a6232a118544d292a3bfca9f" }
+osmosis-std = { git = "https://github.com/osmosis-labs/osmosis-rust", branch = "osmosis-main-clone" }
```
Comment (apollo-pacman) [inline Cargo.toml:15]: Let's set this to latest commit on master
```suggestion
osmosis-std = { rev = "b3df85158eb52ad5d99f3580f4a128bbdaeffa17", git = "https://github.com/osmosis-labs/osmosis-rust.git" }
```

File: src/implementations/junoswap/cw20_stake_external_rewards_msgs.rs [hunk 2]
```diff
@@ -0,0 +1,78 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
```
Comment (apollo-pacman) [inline src/implementations/junoswap/cw20_stake_external_rewards_msgs.rs:1]: Add comment with permalink to original file on github

File: src/implementations/junoswap/cw20_stake_msgs.rs [hunk 3]
```diff
@@ -0,0 +1,121 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
```
Comment (apollo-pacman) [inline src/implementations/junoswap/cw20_stake_msgs.rs:1]: ditto

**Review by apollo-pacman — COMMENTED** [review]
A few comments. why do we need this change? can we not import from daodao?

**Comment by apollo-sturdy** [comment]
Closing. Lets create a new PR once we know which versions of the contracts Junoswap are using.

**Pull request closed without merging**